cgmanifests/cgmanifest.json

```diff
@@ -469,7 +469,17 @@
             },
             "comments": "dlpack"
          }
+      },
+      {
+         "component": {
+            "Type": "other",
+            "Other": {
+               "Name": "SQLite3",
+               "Version": "3.22.0",
```

Comment:
>3.22.0 [](http://example.com/codeflow?start=27&length=6)

we don't actually pin the version elsewhere, do we?

Reply:
Not really, this is just the Ubuntu 18.04 version. Since the target here is component governance, it seems better to list the version we would link to in most cases rather than leave unspecified.